Dockerfile

```diff
@@ -1,5 +1,11 @@
-FROM starefossen/github-pages:137
+# Set to the version for this archive
+ARG VER=v17.03
 
-COPY . /usr/src/app
+# This image comes from the Dockerfile.onbuild file in the docs-builder branch
+# https://github.com/docker/docker.github.io/blob/docs-builder/Dockerfile.onbuild
+FROM docs/docker.github.io:docs-builder-onbuild AS builder
 
-CMD bundle exec jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
\ No newline at end of file
+# Reset the docs:onbuild image, which is based on nginx:alpine
+# This image comes from the Dockerfile in the nginx-onbuild branch
+# https://github.com/docker/docker.github.io/blob/nginx-onbuild/Dockerfile
```

Comment:
agh, this should now be https://github.com/docker/docker.github.io/blob/publish-tools/Dockerfile.nginx.onbuild